users/api/http/transport.go

```diff
@@ -57,13 +60,34 @@ func MakeHandler(svc users.Service, tracer opentracing.Tracer, l log.Logger) htt
 		opts...,
 	))
 
+	mux.Post("/passwd/res-req", kithttp.NewServer(
+		kitot.TraceServer(tracer, "res-req")(passwordResetRequestEndpoint(svc)),
+		decodePasswordResetRequest,
+		encodeResponse,
+		opts...,
+	))
+
+	mux.Post("/passwd/reset", kithttp.NewServer(
```

Comment:
@mteodor I recommend that you use `PATCH /users` here and for the upper one I agree with @manuio, but instead of `password/request` maybe you should put `/password/reset` and for the successful response return `201` because it's part of an async flow.

Reply:
@anovakovic01 Then both endpoints would have the same name

Reply:
@manuio They wouldn't. I've written that the second one should be `PATCH /users` and that the first one should be `POST /password/reset`.

Reply:
@anovakovic01 all clear now. I thought that you were proposing PATCH for both. I agree with you in any case.